app/controllers/helpers/MultipleItemsHelper.scala

```diff
@@ -116,22 +117,26 @@ object MultipleItemsHelper {
     def append(sequence: Seq[A], maybeItem: Option[A]): Seq[A] = maybeItem.foldLeft(sequence)((seq, item) => seq :+ item)
     maybeA.foldLeft(sequence)((sequence, maybeItem) => append(sequence, maybeItem))
   }
+
+  // http://commons.apache.org/proper/commons-lang/javadocs/api-3.6/org/apache/commons/lang3/RandomStringUtils.html
+  // no longer deprecated http://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/RandomStringUtils.html
+  // https://issues.apache.org/jira/browse/LANG-1346
```

Comment:
Nit: This is quite long winded. If this function is no longer depreciated then I think we can use it without a comment (or maybe a smaller one) :wink: